apps/calendar/jsdoc.json

```diff
@@ -5,11 +5,7 @@
   },
   "source": {
     "include": ["apps/calendar/README.md", "apps/calendar/js"],
-    "exclude": ["apps/calendar/js/ext",
```

Comment:
files inside the `ext` folder are not maintained by us, so please keep excluding them. we should not edit these files since we are using volo to download/update these dependencies.